src/tests/test_piranha_java.rs

```diff
@@ -85,3 +84,37 @@ fn test_scenarios_find_and_propagate_panic() {
 
   let _ = execute_piranha(&piranha_arguments);
 }
+
+#[test]
+fn test_user_option_delete_consecutive_lines() {
+  _helper_user_option_delete_consecutive_lines(true);
+}
+
+#[test]
+fn test_user_option_do_not_delete_consecutive_lines() {
+  _helper_user_option_delete_consecutive_lines(false);
+}
+
+fn _helper_user_option_delete_consecutive_lines(delete_consecutive_new_lines: bool) {
+  initialize();
+  let _path = PathBuf::from("test-resources")
+    .join(JAVA)
+    .join(if delete_consecutive_new_lines {
+      "user_option_delete_consecutive_lines"
```

Comment:
I'd still think it would be clearer if we passed these as part of the call to `_helper_user_option_delete_consecutive_lines`, but honestly a matter of taste at this point...